source/modules/.scaffold/scaffold.js

```diff
@@ -1,59 +1,60 @@
-/*!
+import EstaticoModule from '../../assets/js/helpers/module';
+
+const name = '{{name}}';
```

Comment:
Well, only for the instance when constructed, which has a lot of drawbacks (e.g. many instances in the memory, no possibility to use that name in the other static properties and so on)